kitsune/lib/email.py

```diff
@@ -0,0 +1,66 @@
+import logging
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+log = logging.getLogger('k.lib.email')
+RealBackend = import_string(settings.EMAIL_LOGGING_REAL_BACKEND)
+
+
+class LoggingEmailBackend(RealBackend):
```

Comment:
Nevermind--I figured it out. It's because the dev backend is different than what we have in production. That probably means that to land this, we need to change the `settings_local.py` on the servers, right?